_docs/azure_products.md

```diff
@@ -6,6 +6,63 @@ order: 1
+
+# Databases
+## SQL Databases
+Azure offers robust SQL database services. Their primary offering is [Azure SQL Database](https://azure.microsoft.com/en-us/services/sql-database/), and they also offer special versions of the service for [MySQL](https://azure.microsoft.com/en-us/services/mysql/) and [PostgreSQL](https://azure.microsoft.com/en-us/services/postgresql/). Azure SQL databases support all the expected SQL database features as well as built-in machine learning optimization, scaling, high availability, and security. Read the [documentation](https://docs.microsoft.com/en-us/azure/sql-database/) for more information on how to get started. AWS's competing service is [Amazon Relational Database Service (RDS)](https://aws.amazon.com/rds/). GCP's product is named [Cloud SQL](https://cloud.google.com/sql/).
+## NoSQL Databases
+Azure's NoSQL solution is [CosmosDB](https://azure.microsoft.com/en-us/services/cosmos-db/#overview). It offers turnkey distribution, low latency, automatic scaling, and API endpoints compatible with various SQL and NoSQL databases as well as other Azure services. See the [documentation](https://azure.microsoft.com/en-us/services/cosmos-db/#documentation) for more info on everything Cosmos DB can do. AWS offers three slightly different NoSQL services: [DynamoDB](https://aws.amazon.com/dynamodb/), [SimpleDB](https://aws.amazon.com/simpledb/), and [DocumentDB](https://aws.amazon.com/documentdb/). GCP's offering is called [Cloud Datastore](https://cloud.google.com/datastore/).
+
+# Storage
+Azure offers simple object storage via [Azure Blob Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction). Blobs are designed for storing raw data meant to be accessed via cloud and web services, or by other Azure products. It can also be used for storing backups and logging data. Blobs can be accessed via the [Azure Storage REST API](https://docs.microsoft.com/rest/api/storageservices/blob-service-rest-api), Azure command line tools, or libraries available for a variety of languages. AWS's counterpart is the [Amazon Simple Storage Service (S3)](https://aws.amazon.com/s3/). GCP offers similar features with their [Cloud Storage](https://cloud.google.com/storage) product.
```

Comment:
```suggestion
Azure offers simple object storage via [Azure Blob Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction). Blobs are designed for storing raw data meant to be accessed via cloud and web services, or by other Azure products. It can also be used for storing backups and logging data. 

Blobs can be accessed via the [Azure Storage REST API](https://docs.microsoft.com/rest/api/storageservices/blob-service-rest-api), Azure command line tools, or libraries available for a variety of languages. AWS's counterpart is the [Amazon Simple Storage Service (S3)](https://aws.amazon.com/s3/). GCP offers similar features with their [Cloud Storage](https://cloud.google.com/storage) product.
```